tests/e2e/models/Navigation.js

```diff
@@ -24,5 +24,8 @@ class DesktopNav {
     this.zoomIn = bottomToolbar.find('.BRicon.desktop-only.zoom_in');
     this.zoomOut = bottomToolbar.find('.BRicon.desktop-only.zoom_out');
     this.fullScreen = bottomToolbar.find('.BRicon.full');
+    this.searchBox = topToolbar.find('.BRbooksearch.desktop');
+    this.querySign = bottomToolbar.find('.BRquery');
```

Comment:
This name is a little confusing; what's the `querySign`? Is it the magnifying glass icon?